cmd/orchestrator/main.go

```diff
@@ -93,16 +96,16 @@ func run(version, date string) (err error) {
 func runSetup(ctx context.Context, cli cli, sys *system.System) (*task.Orchestrator, error) {
 	c := cli.RunTask
 
+	_ = os.Unsetenv("CIRCLECI_GOAT_CONFIG")
```

Comment:
We unset the configuration env var on top of also stripping it from the task agent command.